src/models/layout_config.rs

```diff
@@ -10,7 +10,7 @@ pub struct LayoutConfig {
 
 impl Default for LayoutConfig {
     fn default() -> Self {
-        let path: PathBuf = [".", "examples", "fonts"].iter().collect();
+        let path: PathBuf = [".", "examples", "fonts", ""].iter().collect();
```

Comment:
Uhh, this is so ugly. It should actually rather be fixed in xelatex. But this is how software engineering works, workarounds wherever you look...

Reply:
Haven't found any better ideas in the docs linked above, and the source code seems to be a macro hell so wanted to avoid diving into it.